users/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+class User(AbstractUser):
+    bio = models.CharField(max_length=256,blank=True)
+    class Meta:
```

Comment:
no need for this meta class, you already set the auth__user_model in settings.py